src/integrationTest/java/org/opensearch/test/framework/cluster/TestRestClient.java

```diff
@@ -109,17 +105,8 @@ public TestRestClient(InetSocketAddress nodeHttpAddress, List<Header> headers, S
         this.sourceInetAddress = sourceInetAddress;
     }
 
-    public HttpResponse get(String path, List<NameValuePair> queryParameters, Header... headers) {
-        try {
-            URI uri = new URIBuilder(getHttpServerUri()).setPath(path).addParameters(queryParameters).build();
```

Comment:
`URIBuilder(...).setPath(path)` has much different behavior for how URLs are constructed that was not aligning well with how `post/delete/...` paths were constructed, URLs would effectively have different rules.  Removing this alternative approach cleans up an inconsistancy in how these test cases work so all REST invocations follow the same path'ing rules.

Reply:
Also, this parameterized constructor wasn't used.  Note; we can always add support for URIBuilder type construction in the future if tests warrant it.